README.md

```diff
@@ -1,6 +1,11 @@
 three.js
 ========
 
+[![Latest NPM release][npm-badge]][npm-badge-url]
+[![License][license-badge]][license-badge-url]
+[![Dependencies][dependencies-badge]][dependencies-badge-url]
+[![Dev Dependencies][devDependencies-badge]][devDependencies-badge-url]
```

Comment:
I actually like `dependencies` because it shows the library has no dependencies. `devDependencies` however, I don't know what it means thus I don't know if it's needed.